src/commands/open.ts

```diff
@@ -22,7 +22,7 @@ const LINKS: Link[] = [
   { name: "Analytics", arg: "analytics", consolePath: "/analytics" },
   { name: "Authentication: Providers", arg: "auth", consolePath: "/authentication/providers" },
   { name: "Authentication: Users", arg: "auth:users", consolePath: "/authentication/users" },
-  { name: "Crash Reporting", arg: "crash", consolePath: "/monitoring" },
+  { name: "Crash Reporting", arg: "crash", consolePath: "/crashlytics" },
```

Comment:
Crashlytics looks like here:
https://console.firebase.google.com/u/8/project/chris-test-project-b7cb4/crashlytics

https://console.firebase.google.com/u/8/project/chris-test-project-b7cb4/monitoring 

redirects to 

https://console.firebase.google.com/u/8/project/chris-test-project-b7cb4/overview